executors/cmdexec/volume.go

```diff
@@ -205,6 +308,35 @@ func (s *CmdExecutor) createAddBrickCommands(volume *executors.VolumeRequest,
 	return commands
 }
 
+func (s *CmdExecutor) createRemoveBrickCommands(volume *executors.VolumeRequest,
+	start, inSet, maxPerSet int, verb string) []string {
+
+	commands := []string{}
+	var cmd string
+
+	// Go through all the bricks and create remove-brick commands
+	for index, brick := range volume.Bricks[start:] {
```

Comment:
FWIW, I find this loop somewhat difficult to read. The way the code flows feels a bit "backwards" to me. I don't have any concrete suggestions at the moment but I wanted to point this out because if now me finds it more challenging a future reader may too.